src/graphql/search/filters/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { Filter, FilterOptions } from '../../../generated/graphql'
+
+// Combine both the filters and return the filters
+export function combineFilters(filters: Array<Filter>): Array<Filter> {
+  return filters.reduce((acc: any, cur: any) => {
```

Comment:
Why are we using `any` types here? It seems like this method has all the information to add types.